src/views/FAQPage/index.vue

```diff
@@ -0,0 +1,106 @@
+<template>
+  <div class="view-container">
+    <div class="data-list faq-data-list">
+      <div
+        v-for="(item, index) in dataItems"
+        :key="index"
+        class="data-item faq-data-item initBorder"
+        :class="{'isClosed': !item.isOpen}"
+      >
+        <FAQ-Item
+          v-bind:question="item.question"
+          v-bind:answer="item.answer"
+          v-bind:toggleAccordion="toggleAccordion"
+        />
+      </div>
+    </div>
+  </div>
+</template>
+
+<script>
+import fetch from 'node-fetch';
+// eslint-disable-next-line import/extensions
+import FAQItem from './FAQItem';
+
+const FAQPage = {
```

Comment:
refactor to Single File Component please

Reply:
Need help with this one too.
NOt sure what `Single File Component`s are. A quick search gives me examples where components are exported directly without being assigned to a variable. Is that what it is or something else?